### PR TITLE
added db restore checklist

### PR DESCRIPTION
**Pull request by bhumilsarvaiya**
## Summary of change
- added checklist for db restoration process using automated backups

### CODE REVIEW

File: v2/contribute/checklists/saas/db-restore.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## Restoring API/Saas related DBs using automated backups
```
Comment (rishabhpoddar) [inline v2/contribute/checklists/saas/db-restore.md:1]:
```suggestion
# Restoring API/Saas related DBs using automated backups
```

File: v2/contribute/checklists/saas/db-restore.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- make sure while restoring, the db instance type, security groups and all other configuration (including the root password) are same as the old db cluster
+- update the route 53 entry related to this db (production-mysql.supertokens.dns)
+- once the restoration is finished, update the following:
+    - production node api server config
+    - db configuration on retool site
+    - db configuration on metabase site
+- restart production node api server containers
+- remove old cluster
+
+### Database - API Saas DB
```
Comment (rishabhpoddar) [inline v2/contribute/checklists/saas/db-restore.md:27]: What does this heading mean?

File: v2/contribute/checklists/saas/db-restore.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+- get the latest automated backup of the saas db
+- use restore snapshot option
+- make sure while restoring, the db instance type, security groups and all other configuration (including the root password) are same as the old db cluster
+- update the route 53 entry related to this db (production-saas-users-mysql.supertokens.dns)
+- once the restoration is finished, update the following:
+    - production node api server config
+- restart production node api server containers
+- remove old cluster
+
```
Comment (rishabhpoddar) [inline v2/contribute/checklists/saas/db-restore.md:37]: No change to the sidebar? How did you even test this?!

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: v2/contribute/checklists/saas/db-restore.md (hunk 3 above)
Comment (bhumilsarvaiya) [reply v2/contribute/checklists/saas/db-restore.md:37]: What do you mean by sidebar?

File: v2/contribute/checklists/saas/db-restore.md (hunk 2 above)
Comment (bhumilsarvaiya) [reply v2/contribute/checklists/saas/db-restore.md:27]: saas users db used to store active users

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**